trajopt/src/kinematic_terms.cpp

```diff
@@ -157,48 +239,33 @@ void CartPoseErrCalculator::Plot(const tesseract_visualization::Visualization::P
 
 MatrixXd CartPoseJacCalculator::operator()(const VectorXd& dof_vals) const
```

Comment:
This should be updated to look similar to TrajOpt Ifopt `CartPosConstraint::CalcJacobianBlock` the issue below is that calcTransformError should not be used for jacobian calculation because it normalizes the rotational error [-pi, pi]. See `tesseract_common::calcRotationalError2`

Reply:
It's probably a good idea to add a new function to tesseract_common like `calcTransformErrorJac' and update the documentation for `calcTransformError` to have a doxygen warning stating it should not be used for numerical jacobians.

Reply:
I created this [PR](https://github.com/tesseract-robotics/tesseract/pull/971) adding the new function.

Reply:
Updated, will have to re-run the CI after that PR is merged in.